tests/sampling/test_mcmc_external.py

```diff
@@ -81,7 +81,8 @@ def test_step_args():
             nuts_sampler="numpyro",
             target_accept=0.5,
             nuts={"max_treedepth": 10},
-            random_seed=1410,
+            random_seed=1411,
```

Comment:
What's the story with the seeds? Is the test fragile on purpose?

Reply:
It started failing on the CI, perhaps numpyro changed something in the adaptation and doesn't try to respect target accept as much. It passed locally for me no matter what versions I tried to isntall. It's definitely unrelated to this PR so I'm not worried